python/lsst/ctrl/bps/wms/panda/conf_example/example_panda.yaml

```diff
@@ -1,18 +1,24 @@
-pipelineYaml: ${CTRL_BPS_DIR}/python/lsst/ctrl/bps/wms/panda/conf_example/HSC-PANDA.yaml
+pipelineYaml: /opt/lsst/software/HSC-PANDA.yaml
 project: dev
 campaign: "configuration_example"
 submitPath: ${HOME}/submit/{outCollection}
-container_obs_panda_edge_node_dir: /opt/lsst/software/stack/stack/miniconda3-py38_4.9.2-0.6.0/Linux64/ctrl_bps/21.0.0-18-gf2cd492+6c749b2ca5/python/lsst/ctrl/bps/wms/panda/edgenode
-container_CTRL_MPEXEC_DIR: /opt/lsst/software/stack/stack/miniconda3-py38_4.9.2-0.6.0/Linux64/ctrl_mpexec/21.0.0-30-g82f2559+c64cb64c6b/
 
 computing_cloud: LSST
+
+#This setting supersede cloud and memory requirements
+#computeSite: DOMA_LSST_GOOGLE_TEST_HIMEM
+
 maxwalltime: 90000
 requestMemory: 2000
 maxattempt: 1
 
 whenSaveJobQgraph: "NEVER"
```

Comment:
Just a heads up, planning on making this the default value in the execution butler version so it won't need to be here too much longer.